spec/models/bus_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Bus, type: :model do
```

Comment:
please check if we can skip this explicit mention of `type`

Reply:
removed the explicit mention of `type: :model`. Thanks!